cypress/e2e/create-password.cy.js

```diff
@@ -0,0 +1,254 @@
+describe("create password page", () => {
+  beforeEach(() => {
+    cy.visit("/create-password");
+  });
+
+  it("shows the correct page title", () => {
+    cy.get('[data-cy="section-heading-h1"]').contains(
+      "Create a password for your account"
+    );
+  });
+
+  it("can go back to the previous page", () => {
+    cy.get('[data-cy="back-link"]').click();
+    cy.url().should("eq", "http://localhost:3000/email-address");
+  });
+
+  it("shows the password length requirements", () => {
+    cy.contains("must contain 8 digits");
+  });
+
+  it("shows the password input field and value", () => {
+    cy.get('[data-cy="password-input"]')
+      .type("passwordtest")
+      .should("have.value", "passwordtest");
+  });
+
+  it("shows the label for the confirm password input", () => {
+    cy.contains("Confirm password");
+  });
+
+  it("shows the password input field and value", () => {
```

Comment:
Maybe you could change this test, so it shows that you're testing the second field. Or maybe you can put your tests in `context` blocks to make it clear which elements your tests relate to.